src/Core/Checkout/Cart/Event/LineItemAddedFinishEvent.php

```diff
@@ -0,0 +1,63 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Checkout\Cart\Event;
+
+use Shopware\Core\Checkout\Cart\Cart;
+use Shopware\Core\System\SalesChannel\SalesChannelContext;
+use Symfony\Contracts\EventDispatcher\Event;
+
+/**
+ * @deprecated tag:v6.4.0 - Will implement Shopware\Core\Framework\Event\ShopwareSalesChannelEvent
```

Comment:
copy paste :smile: 

there is no need to do this on new events. please remove the deprecations completely and implement the interface correctly